ui/cypress/integration/volume/volumelist.spec.js

```diff
@@ -0,0 +1,67 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+});
+
+// Navigation tests
+describe('Volume list', () => {
+  beforeEach(() => {
+    // Visit is automatically prefixed with baseUrl
+    cy.visit('/volumes');
+    cy.stubHistory();
```

Comment:
If you `visit()` in `beforeEach`, make sure you don't `visit()` again within each test case, or it may create issues with the test (some network requests would be interrupted) - see [`07a34b5` (#2891)](https://github.com/scality/metalk8s/pull/2891/commits/07a34b5af5df07fbfa95f08408a93f3fff7eb043)